src/SearchParameters/index.js

```diff
@@ -272,7 +272,7 @@ function SearchParameters(newParameters) {
   /**
    * Enable the ranking informations in the response
    * @see https://www.algolia.com/doc#getRankingInfo
-   * @member {integer}
+   * @member {number}
```

Comment:
Well since `{integer}` is not official, yes. But maybe we could just do the obvious: specify `{integer}` even if it's not a JS type. It still what the user needs to know.

We should feed the user with valuable information instead of trying to fit into jsdoc?

Or maybe a comment